frontend/src/scenes/authentication/redirectToLoggedInInstance.ts

```diff
@@ -34,12 +35,29 @@ const SUBDOMAIN_TO_NAME = {
     app: 'US',
 } as const
 
+export function cleanedCookieSubdomain(loggedInInstance: string | null): string | null {
+    try {
+        // replace '"' as for some reason the cookie value is wrapped in quotes e.g. "https://eu.posthog.com"
+        const url = loggedInInstance?.replace(/"/g, '')
```

Comment:
since we know the loggedInInstance we care about should be one of `app|eu.posthog.com` maybe worth explicitly checking for that here instead?

Prevents other issues, like `"` being a valid url character 💀 , and not redirecting to an attack instance if _someone_ manages to edit the cookie?

Don't think we need to support this for self hosted instances, since there shouldn't be eu/us confusion there

Reply:
Yeah, I was thinking about that as I changed it... wanted to fix the literal issue in case it was related to the incident... but may as well while it's stolen our focus